.github/issue_template.md

```diff
@@ -0,0 +1,36 @@
+<!---
+ Please use this template to fill your issue, issues that does not
+ follow this will be asked to be edited to match the template, or will be closed
+
+ We use github issues for problem/bug reports, for questions, please use the Discord channel, the link for it can be found on the project README
+-->
+
+## Description
+
+Describe your issue here
+
+## Development enviroment
+
+Flame version:
+
+<!--
+NOTE: Flame supports stable and beta channel of Flutter, issues out of those channels may not be prioritized, or closed.
+-->
+
+Flutter doctor:
+
+```
+ Include your flutter doctor output here
+```
+
+## Runtime
+
+This issue is related to running on which platform? (Select all that apply)
+
+ - [ ] Android :android:
+ - [ ] iOS :apple:
+ - [ ] Web :browser:
+
+## Minimal reproducible code (Required for bugs)
+
+Include a minimal code that produces the problem, add it inline using code block markdown, or a link to github repository:
```

Comment:
```suggestion
Include a minimal code that produces the problem (add it inline using code block markdown), or a link to Github repository:
```

Reply:
If we need to capitalize it correctly it should be GitHub ;)